rotkehlchen/exchanges/binance.py

```diff
@@ -593,24 +626,28 @@ def query_lending_interests_history(
             start_ts: Timestamp,
             end_ts: Timestamp,
     ) -> bool:
-        """Queries Binance lending interest history, transforms it into `HistoryEvent` objects and saves it in the database.
+        """Queries Binance Simple Earn history, transforms it into `HistoryEvent` objects
+        and saves it in the database.
 
-        "DAILY" -> flexible savings.
-        "CUSTOMIZED_FIXED" -> fixed savings.
-        "ACTIVITY" -> fixed savings but as a result of token partners.
+        "BONUS" -> Bonus tiered APR.
+        "REALTIME" -> Real-time APR.
+        "REWARDS" -> Historical rewards.
 
         `end_ts` - `start_ts` <= 30days. This is handled using `_api_query_list_within_time_delta`
-        using `BINANCE_LENDING_INTEREST_HISTORY_INTERVAL_TS` as the timedelta.
+        using `API_TIME_INTERVAL_CONSTRAINT_TS` as the timedelta.
 
         Lending Interest History Documentation:
-        https://binance-docs.github.io/apidocs/spot/en/#get-interest-history-user_data-2
+        https://binance-docs.github.io/apidocs/spot/en/#get-flexible-rewards-history-user_data
+        https://binance-docs.github.io/apidocs/spot/en/#get-locked-rewards-history-user_data
 
         May raise:
         - RemoteError
         - BinancePermissionError
-        """  # noqa: E501
+        """
         ranges = DBQueryRanges(self.db)
-        range_query_name = f'{self.location}_lending_history_{self.name}'
+
+        # Query and save flexible simple earn history
+        range_query_name = f'{self.location}_flexible_lending_history_{self.name}'
```

Comment:
Don't change the name here. We already will have `'{self.location}_lending_history` in the DB. So we need to keep same name.

Unless it's a completely different one.

Or if you want to keep the same, or if it's a different one then you need to do a DB upgrade renaming or removing the previous key values.

Reply:
[Found it](https://www.binance.com/en/support/announcement/introducing-simple-earn-the-evolution-of-binance-savings-binance-staking-b067f764f4f14e5aa46cca1bf2b55ab8)
TL;DR: Simple Earn is a rebranding of the same thing.
So, we can use the same range_query_name